staging/src/k8s.io/apiserver/pkg/server/options/admission.go

```diff
@@ -84,7 +89,13 @@ func (a *AdmissionOptions) AddFlags(fs *pflag.FlagSet) {
 		"The names in the below list may represent a validating plugin, a mutating plugin, or both. "+
 		"Within each phase, the plugins will run in the order in which they are passed to this flag. "+
 		"Comma-delimited list of: "+strings.Join(a.Plugins.Registered(), ", ")+".")
-
+	fs.MarkDeprecated("admission-control", "Use --enable-admission-plugin or --disable-admission-plugin instead. Will be removed in a future version.")
+	fs.StringSliceVar(&a.EnabledAdmissionPlugins, "enable-admission-plugin", a.EnabledAdmissionPlugins, ""+
```

Comment:
This is unclear… if you specify this, is it in addition to default-enabled plugins, or does it override?

Reply:
Ah, the flag name ("enable") is better than the variable name ("enabled"). Should still clarify in the help that this is in addition to any enabled-by-default plugins

Reply:
OK